bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/config/TestBigtableVeneerSettingsFactory.java

```diff
@@ -133,8 +134,15 @@ private void initializeClients() throws IOException{
   }
 
   /**
-   * This test runs only if it finds "test.client.project.id" & "test.client.project.id"
-   * VM arguments. Then it calls to an actual Bigtable Table & performs the checks below:
+   * This test runs only if it founds below env variables,
+   * Then it calls to an actual Bigtable Table & performs the checks below:
```

Comment:
Please don't use `&`, since the comments are valid HTML.  Please use the word `and`